src/storage.ts

```diff
@@ -290,10 +290,14 @@ export class Storage extends Service {
    * @param {StorageOptions} [options] Configuration options.
    */
   constructor(options: StorageOptions = {}) {
+    let url = `https://${options.apiEndpoint}/storage/v1`
+    if (process.env.STORAGE_EMULATOR_HOST != undefined) {
```

Comment:
I think we can pass this in to the constructor, like so:

```js
const storageClient = new Storage({
  baseUrl: process.env.STORAGE_EMULATOR_HOST
})
```

Reply:
I don't believe we expose `baseUrl` as a configurable property (I could be wrong)